risc0/build/src/lib.rs

```diff
@@ -63,14 +63,18 @@ pub struct GuestListEntry<'a> {
     pub path: &'a str,
 }
 
+/// Represents a RISC Zero method
 #[derive(Debug)]
-struct Risc0Method {
-    name: String,
-    elf_path: PathBuf,
+pub struct Risc0Method {
```

Comment:
Why make both `Risc0Method` and `GuestListEntry` public? Don't they contain the same information? `GuestListEntry` looks more complete in this regard.